README.md

```diff
@@ -133,33 +132,55 @@ mongo --version
 
 Additonally, there is an `.nvmrc` file in the root of the project. You can use [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm) to make sure you are using the right version of node for this and other projects! This also ensures that any cloud deployments of the project use the same version of Node.
 
-Getting a local instance of Quill up and running takes less than 5 minutes! Start by setting up the database. Ideally, you should run MongoDB as a daemon with a secure configuration (with most linux distributions, you should be able to install it with your package manager, and it'll be set up as a daemon). Although not recommended for production, when running locally for development, you could do it like this
+We use `dotenv` to keep track of environment variables, so be sure to stop tracking the `.env` file in Git:
 
 ```bash
-mkdir db
-mongod --dbpath db --bind_ip 127.0.0.1
+git update-index --assume-unchanged .env
 ```
 
-Install the necessary dependencies:
+After doing this, fill in the environment variables in the `.env` before running Quill.
+
+### Local Deployment
+
+#### MongoDB
+
+Ideally, you should run MongoDB as a daemon with a secure configuration (with most linux distributions, you should be able to install it with your package manager, and it'll be set up as a daemon). Although not recommended for production, when running locally for development, you could do it like this
 
 ```bash
-npm install
+mkdir db
+mongod --dbpath db --bind_ip 127.0.0.1
 ```
 
-We use `dotenv` to keep track of environment variables, so be sure to stop tracking the `.env` file in Git:
+#### SMTP
+
+This step is optional and is only required if you want to test the email-related functionality of Quill. The easiest option is to use the SMTP server provided by your personal email (Gmail, Outlook, etc.). Look for the documentation about SMTP for your respective email and fill in the values in the `.env` accordingly. Be warned that sending many emails this way is not recommended and this method should only be used for testing.
```

Comment:
One note here for Gmail specifically is that you'll need to enable "less secure apps". https://support.google.com/accounts/answer/6010255?hl=en

Reply:
Right, I will add that